### DISCUSSION TITLE
Looking to Hire an Engineer

### DISCUSSION THREAD

**Discussion by MIDIculous** [ant-media/Ant-Media-Server#5585] [Freelancing]
We are looking to hire someone to get our Ant Media Server installed on our Linux machine.

**Comment by congkuan** [comment]
hello,I can help you.

**Reply by MIDIculous** [reply]
I sent you an email

**Reply by mekya** [reply]
Great to see this kind of collaboration happening. Good Luck guys.

**Comment by Prateekstupa** [comment]
hey I can help you in setup and maintaining

**Reply by MIDIculous** [reply]
I already got some help from someone on Upwork, but thanks.
